samcli/commands/local/cli_common/options.py

```diff
@@ -48,7 +48,16 @@ def local_common_options(f):
             default=False,
             help="If set, will emulate a shutdown event after the invoke completes, "
             "in order to test extension handling of shutdown behavior.",
-        )
+        ),
+        click.option(
+            "--container-host",
```

Comment:
For any security reasons, do we need to validate if this option valid?

Reply:
This is a good point. Let me check on this and get back to you.

Reply:
For security reasons, there is nothing we can validate. There is a use case that docker daemon is running on a different host and the customer wants to use that host. We won't be able to understand if it's a good host provided by customer or just a bad actor.